### PR TITLE
Katie pillar page

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

**Comment by kcbreuil** [comment]
@asasmith [](url) I am waiting on the pillarCardComponent to be merged to add underneath the pillardescription component
Screen Shot 2021-04-12 at 11 51 59 PM

**Comment by asasmith** [comment]
This is for the `main` branch. Can you update for our staging branch?

File: src/components/eventHero.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import tw from 'twin.macro';
 
 import EventCard from './eventCard';
 
-const HeroWrapper = styled.section`
+const HeroWrapper = styled.div`
```
Comment (asasmith) [inline src/components/eventHero.js:10]: Why change this to a `div`?

File: src/components/pillarComponent.js [hunk 2]
```diff
@@ -5,16 +5,16 @@ import styled from 'styled-components';
 import tw from 'twin.macro';
 
 const StyledSection = styled.section`
-    ${tw`min-h-3/4-screen flex flex-col justify-center pb-8 text-white`}
+    ${tw` bg-white transform -translate-y-20 md:-translate-y-1/4 flex flex-col justify-center mx-10`}
```
Comment (asasmith) [inline src/components/pillarComponent.js:8]: Remove space in front of `bg-white`

**Review by asasmith — CHANGES_REQUESTED** [review]

File: src/components/eventHero.js (hunk 1 above)
Comment (kcbreuil) [reply src/components/eventHero.js:10]: @asasmith a section wraps these two components. we would have a section and a div inside of a section. Now it's two divs inside of a section.

Comment (asasmith) [reply src/components/eventHero.js:10]: Am I looking at this wrong? It looks like it's a "top level" component.
https://github.com/SilveradoPolicy/silverado/blob/1ba8d9d4f7abb2ff39a5a34be7481a0651357a71/src/pages/event-template.js#L66-L69

Comment (kcbreuil) [reply src/components/eventHero.js:10]: @asasmith Okay, there are two eventhero components-- the one you are highlighting here lives in the event folder under components. when I created this one I did not see this folder. These are two different components performing differently. The eventhero I changed is the one that lives under components and will be rendered in the eventpage under templates. I am going to move this out of there

**Review by asasmith — APPROVED** [review]

**Pull request merged**